src/Utils.cc

```diff
@@ -133,3 +133,19 @@ utils::firstChar(const QString &input)
 
         return QString::fromUcs4(&input.toUcs4().at(0), 1).toUpper();
 }
+
+QString
+utils::humanReadableFileSize(const qlonglong bytes)
+{
+        constexpr static const char *units[] = {"B", "KB", "MB", "GB", "TB"};
```

Comment:
Use the correct [IEEE 1541](https://en.wikipedia.org/wiki/IEEE_1541-2002) units: KiB, MiB, GiB, TiB, etc